Paralayout/UIView+Distribution.swift

```diff
@@ -43,18 +43,18 @@ public enum HorizontalDistributionAlignment {
     /// Align to the leading edge, inset by the specified amount.
     ///
     /// - inset: An inset from the leading edge towards the center of the distribution axis.
```

Comment:
nit: `parameter` (don't feel strongly with/without it, just want to make sure we're consistent)

Reply:
Thanks, missed earlier but fixed now 🙏